ui/src/main/kotlin/edu/stanford/spezi/ui/AsyncButton.kt

```diff
@@ -19,13 +19,10 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.alpha
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.graphics.Shape
-import edu.stanford.spezi.ui.Colors
-import edu.stanford.spezi.ui.ComposableContent
-import edu.stanford.spezi.ui.ComposeValue
-import edu.stanford.spezi.ui.Sizes
-import edu.stanford.spezi.ui.SpeziTheme
-import edu.stanford.spezi.ui.ThemePreviews
-import edu.stanford.spezi.ui.testing.testIdentifier
+import edu.stanford.spezi.ui.theme.Colors
+import edu.stanford.spezi.ui.theme.Sizes
+import edu.stanford.spezi.ui.theme.SpeziTheme
+import edu.stanford.spezi.ui.theme.ThemePreviews
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.launch
```

Comment:
I was hesitant so far to copy over this component, since I'd usually assume that `AsyncTextButton` is the `TextButton` with async functionality, which isn't exactly the case, since it provides some other defaults as the `TextButton` has

In general, we may use a similar approach here as we planned to use for the TextFields (i.e. a shared interface with some styling parameter to allow for easier wrapping across all types of textfields/buttons), so happy to hear your thoughts!